affiliatewp-order-details-for-affiliates.php

```diff
@@ -89,46 +113,49 @@ public static function instance() {
 			self::$instance->includes();
 			self::$instance->hooks();
 
-			self::$instance->order_details = new AffiliateWP_Order_Details_For_Affiliates_Order_Details;
-			self::$instance->emails        = new AffiliateWP_Order_Details_For_Affiliates_Emails;
-			self::$instance->shortcodes    = new AffiliateWP_Order_Details_For_Affiliates_Shortcodes;
+			self::$instance->order_details = new AffiliateWP_Order_Details_For_Affiliates_Order_Details();
+			self::$instance->emails        = new AffiliateWP_Order_Details_For_Affiliates_Emails();
+			self::$instance->shortcodes    = new AffiliateWP_Order_Details_For_Affiliates_Shortcodes();
```

Comment:
```suggestion
			self::$instance->order_details = new AffiliateWP_Order_Details_For_Affiliates_Order_Details;
			self::$instance->emails        = new AffiliateWP_Order_Details_For_Affiliates_Emails;
			self::$instance->shortcodes    = new AffiliateWP_Order_Details_For_Affiliates_Shortcodes;
```
Leave parentheses off empty instantiations.